library/unwind/Cargo.toml

```diff
@@ -32,3 +32,5 @@ llvm-libunwind = []
 # If crt-static is enabled, static link to `libunwind.a` provided by system
 # If crt-static is disabled, dynamic link to `libunwind.so` provided by system
 system-llvm-libunwind = []
+
+android-libunwind = []
```

Comment:
Could you reuse the `system-llvm-libunwind` feature instead of introducing a new feature?